include/flashinfer/attention/mla_hopper.cuh

```diff
@@ -698,10 +732,12 @@ __global__ __launch_bounds__(KTraits::NUM_THREADS) void BatchMLAPageAttentionHop
 #pragma unroll 1
       for (; kv_tile_idx >= 0; --kv_tile_idx) {
         pipeline_kv.producer_acquire(smem_pipe_write_kv);
-        load_kv<KTraits>(&smem_storage, ckv, kpe, kv_indices, ckv_stride_n, ckv_stride_page,
-                         kpe_stride_n, kpe_stride_page, kv_bound,
-                         block_iter_base + kv_tile_idx * CTA_TILE_KV, block_size,
-                         smem_pipe_write_kv.index());
+        PROFILER_EVENT_START(variant, ProfileEventType::kIssueLoadKV);
+        load_kv<false, KTraits>(&smem_storage, ckv, kpe, kv_indices, ckv_stride_n, ckv_stride_page,
+                                kpe_stride_n, kpe_stride_page, kv_bound,
+                                block_iter_base + kv_tile_idx * CTA_TILE_KV, block_size,
+                                smem_pipe_write_kv.index());
+        PROFILER_EVENT_END(variant, ProfileEventType::kIssueLoadKV);
```

Comment:
What gets measured by the event `ProfileEventType::kIssueLoadKV` time it takes to load kv data (i.e. move the data from GMEM->SMEM) using async TMA / cp.async loads or the instruction issue time?

Reply:
The measurement only captures the instruction issue time; to determine the GMEM-to-SMEM time, calculate the interval between the kIssueLoadKV event’s start timestamp and that of its consumer.

Reply:
Thanks for confirm.